### PR TITLE
fix(repo): update release-please config

### PR DESCRIPTION
**Pull request by RogerLamTd**

### CODE REVIEW

**Comment by dantaik** [comment]
how will this PR impact other pull requests?

**Comment by RogerLamTd** [comment]
> how will this PR impact other pull requests?

pull requests from other scopes will now be included in the release-please changelog, it shouldn't affect other PRs

**Review by dionysuzx — APPROVED** [review]

File: .github/workflows/repo--release-please.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
   release-please:
     runs-on: [taiko-runner]
     steps:
-      - uses: google-github-actions/release-please-action@v4
+      - uses: googleapis/release-please-action@v4
         with:
           token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
+          config-file: release-please-config.json
```
Comment (d1onys1us) [inline .github/workflows/repo--release-please.yml:19]: Do we need this or it already finds it by default?

Comment (RogerLamTd) [reply .github/workflows/repo--release-please.yml:19]: can remove, will push

**Review by dantaik — APPROVED** [review]

**Pull request merged**